galasa-extensions-parent/dev.galasa.ras.couchdb/src/test/java/dev/galasa/ras/couchdb/internal/CouchdbValidatorImplTest.java

```diff
@@ -71,6 +71,227 @@ public MockCloseableHttpResponse getResponse() {
         }
     }
 
+    public static class checkDatabasePresentInteraction extends BaseHttpInteraction {
```

Comment:
Classes should be PascalCase - same applies throughout this file

Also, there's a lot of duplicate code for these interactions, so you could perhaps subclass some and override the bits that you want to change?